### PR TITLE
fix missing dependencies for successfully installation redisai #23

### PR DESCRIPTION
**Pull request by sandroci**
As detailed described in the issue #23 the installation of redisai in a complete new environment did fail because of missing dependencies. Therefore I have included those deps in the `setup.py` and removed the unnecessary import checks in the related files (`client.py` and `tensor.py`). 

I have created a new python virtual environment and installed redisai again. Everything worked as expected and no more missing dependencies.

### CODE REVIEW

File: redisai/tensor.py [hunk 1]
```diff
@@ -78,22 +74,22 @@ def __init__(self,
...
-                blobarr += b
-            size = len(blobs)
-            ret_blobs = bytes(blobarr)
-            shape = [size] + list(shape)
-        else:
-            ret_blobs = bytes(blobs[0])
+                    blobarr += b
+                    size = len(blobs)
+                    ret_blobs = bytes(blobarr)
+                    shape = [size] + list(shape)
```
Comment (hhsecond) [inline redisai/tensor.py]: I am not sure how far we could rely on autoformattes - This `for loop` is a typical example

Comment (sandroci) [reply redisai/tensor.py]: @hhsecond I have removed all the auto-formatter changes.

**Comment by hhsecond** [comment]
@sandroci I had to push my changes before this PR, which had introduced few conflicts. Can you please fix those?

**Comment by sandroci** [comment]
@hhsecond sorry for the extreme late response. For unknown reason I didn't received a notification about your comment. No worries, I will fix it asap.

**Comment by hhsecond** [comment]
@sandroci I don't seem to have edit access to this PR. Can you fix these conflicts anytime soon? I am planning an internal revamp to clean up the bloat and it'll create more conflicts. I'd like to merge it before that

**Comment by hhsecond** [comment]
Closing because of inactivity. Feel free to re-open

**Pull request closed without merging**